src/_includes/partials/givefeedback.njk

```diff
@@ -0,0 +1,26 @@
+<section data-pagefind-ignore class="py-100 mt-400 mb-600 {% if removeBorder %} {% else %}by-sm b-dark {% endif %}">
+  <h2 class="mt-300 mb-400">
+    {{ givefeedback[locale].heading }}
+  </h2>
+
+  <p class="mb-400">
```

Comment:
Can we change `mb-400` to `mb-300` please to match the new spacing classes?

Reply:
```suggestion
  <p class="mb-300">
```

Reply:
Done!